pytorch/training/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -155,7 +155,7 @@ RUN mkdir /tmp/openmpi \
  && cd / \
  && rm -rf /tmp/openmpi
 
-ENV PATH="$OPEN_MPI_PATH/bin:$PATH"
+ENV PATH="$OPEN_MPI_PATH/bin:$EFA_PATH/bin:$PATH"
```

Comment:
Yes, we did, but the EFA_PATH doesn't contain any binaries that would be used for actual customer jobs. It only contains basic utility binaries that can still be used by passing the full path to it, like `/opt/amazon/efa/bin/fi_info` instead of just `fi_info`.